spec/lib/wisper/broadcasters/send_broadcaster_spec.rb

```diff
@@ -10,7 +10,7 @@ module Broadcasters
 
           it 'sends event to listener without any arguments' do
             expect(listener).to receive(event).with(no_args())
-            subject.broadcast(listener, anything, event, args)
+            subject.broadcast(listener, anything, event, *args)
```

Comment:
Given the title of the spec should this be this instead, where we don't include the args parameter at all? 

```suggestion
            subject.broadcast(listener, anything, event) # leaves out args entirely
```

That may be best done in a different test though?